src/hifz/dataserver.py

```diff
@@ -0,0 +1,27 @@
+import csv
+from pathlib import Path
+
+from hifz.models import Card
+
+
+class DataServer:
+    def read_entries(self, file_path: str) -> list[Card]:
+        """Reads the entries associatied with the csv at file_path.
+
+        Args:
+            file_path (str): The file path of the csv.
+
+        Returns:
+            list[Card]: The list of cards.
+        """
+        cards = []
+
+        try:
+            with Path(file_path).open() as f:
+                entries = csv.DictReader(f)
+                cards = [Card(e["front"], e["back"]) for e in entries]
```

Comment:
I wonder if we can return cards in here. I'm not sure about the order of operations with the with statement and try and except.

Reply:
I think returning at the end is fine. It feels more idiomatic in my mind. I tend to keep exception handling blocks short, but I think in this particular case it can be done either way